examples/ansible/setup_grafana.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Grafana on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment of Grafana on a SUSE ALP Dolomite host.
+# The steps include fetching the Grafana image, setting up the Grafana container, and providing access information.
+# The steps are based on https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-grafana-with-podman
+
+- name: Setup ALP system for Grafana
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: grafana
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/grafana:latest
```

Comment:
Again, I see no point of defining static content as variables.

Reply:
My bad, I've followed the pattern for consistency, I'll make the required changes.

Reply:
Consistency is good argument. Sorry, my bad.
Feel free to put it back.